Dockerfile

```
@@ -0,0 +1,20 @@
+# Use an official Node.js runtime as a parent image
+FROM node:22
+
+# Set the working directory in the container
+WORKDIR /usr/src/app
+
+# Create the cache directory
+RUN mkdir -p ./cache
+
+# Copy the current directory contents into the container at /usr/src/app
+COPY . .
+
+# Install any needed packages specified in package.json
+RUN npm install
+RUN npm update
+# Define environment variable
+ENV NODE_ENV production
```

This is the old syntax.  New syntax uses "=".

```suggestion
ENV NODE_ENV=production
```